apps/web/src/app/(signing)/sign/[token]/sign-dialog.tsx

```diff
@@ -9,6 +9,15 @@ import {
   DialogTrigger,
 } from '@documenso/ui/primitives/dialog';
 
+const truncateTitle = (title: string, maxLength: number) => {
```

Comment:
I'm thinking we should extract this in a place like `/web/src/helpers` for example, so it can re-used if necessary.

Reply:
@catalinpit Sure, I'm on it !